index.html

```diff
@@ -0,0 +1,176 @@
+					<p>Logout</p>
+					<p>My Page</p>
+					<p>English</p>
+				</div>
+			</div>
+			<div class="menu" id="mobile-menu">
+				<div class="menu-bottom">
+					<img src="./images/logo.svg" alt="" />
+					<div class="left-menu">
+						<ul class="left">
+							<li><a class="active" href="#">Home</a></li>
+							<li><a href="./about.html">About</a></li>
+							<li><a href="">Program</a></li>
+							<li><a href="">News</a></li>
+						</ul>
+						<button>CC compaign</button>
+					</div>
+				</div>
+			</div>
+			<div id="humburger">
+				<span class="material-symbols-outlined"> menu </span>
+			</div>
+		</header>
+		<section id="headline">
+			<div class="content">
+				<div class="heading">
+					<h3>“Hello! Sharing world”</h3>
+					<h1>2021 Glocal Mission Summit</h1>
+				</div>
+				<div class="discription">
+					<p>
+						The Glocal Mission Summit is a biannual event where
+						people engage in conversations around mission and
+						imagine innovative ways to engage and embrace our local
+						communities that are becoming increasingly diverse.
+					</p>
+				</div>
+				<div class="location">
+					<h4>2021.10.15(THU) ~ 16(FRI)</h4>
+					<p>@ National Museum of Korea, Art Center Nabi and more</p>
+				</div>
+			</div>
+		</section>
+		<section id="mainProgram">
+			<div class="program">
+				<div class="title">
+					<h4>Main Program</h4>
+					<hr />
+				</div>
+				<div class="cards">
+					<div class="card">
+						<img
+							src="./images/webhook-fill-0-wght-200-grad-0-opsz-481.svg"
+							alt=""
+						/>
+						<h5>Jerusalem</h5>
+						<p>
+							Sent ad jesus to be an international presence in our
+							neighborhoods (90 min)
+						</p>
+					</div>
+					<div class="card">
+						<img
+							src="./images/webhook-fill-0-wght-200-grad-0-opsz-481.svg"
+							alt=""
+						/>
+						<h5>Judea</h5>
+						<p>
+							Engaging with the questions and challenges in our
+							broader context (90 min)
+						</p>
+					</div>
+					<div class="card">
+						<img
+							src="./images/webhook-fill-0-wght-200-grad-0-opsz-481.svg"
+							alt=""
+						/>
+						<h5>Samaria</h5>
+						<p>
+							Learning to listen and grow alongside people
+							different than ourselves (120 min)
+						</p>
+					</div>
+					<div class="card">
+						<img
+							src="./images/webhook-fill-0-wght-200-grad-0-opsz-481.svg"
+							alt=""
+						/>
+						<h5>Ends of the earth</h5>
+						<p>
+							Returning to our daily life & work with new eyes/ hearts, sent by the spirit to witness “glocally”
+						</p>
+					</div>
+					<div class="card">
+						<img
+							src="./images/webhook-fill-0-wght-200-grad-0-opsz-481.svg"
+							alt=""
+						/>
+						<h5>Lecture</h5>
+						<p>
+							Listen to the speaker from various countries about
+							the message of sharing and opening
+						</p>
+					</div>
+				</div>
+				<div class="seemore">
+					<p>See the whole program</p>
+				</div>
```

Comment:
- [ ] According to the design, this button looks different on desktop and mobile screens. Currently, your mobile design for the button doesn't match the given design. I'll suggest you update this also.

- - Your current mobile design:
![Screenshot from 2023-06-29 14-21-50](https://github.com/sheezu900/module1-capstone-project/assets/107632780/f86eef6f-d236-42e5-bd72-204564018ed3)

- - Given design:
![Screenshot from 2023-06-29 14-22-22](https://github.com/sheezu900/module1-capstone-project/assets/107632780/dc98d20f-07a3-4cf2-8a77-1c713d898f26)